src/prefect/variables.py

```diff
@@ -0,0 +1,41 @@
+import sys
+from types import ModuleType
+from typing import Optional
+
+from prefect._internal.concurrency.api import create_call, from_sync
+from prefect.client.orchestration import get_client
+
+
+def get(name: str, default: str = None) -> Optional[str]:
+    """Get a variable by name from a sync context"""
+    variable = get_variable_by_name(name)
+    return variable.value if variable else default
+
+
+async def aget(name: str, default: str = None) -> Optional[str]:
```

Comment:
@madkinsz is this what you had in mind when you said offering something that users could await?

Reply:
Technically it should probably use `sync_compatible` — we don't really publish two interfaces as a general rule.

Reply:
duh 🤦‍♂️, good call